libcxx/test/std/language.support/support.types/byte.pass.cpp

```diff
@@ -10,26 +10,34 @@
 #include <type_traits>
 #include "test_macros.h"
 
-// XFAIL: c++03, c++11, c++14
+// REQUIRES: std-at-least-c++17
 
 // std::byte is not an integer type, nor a character type.
-// It is a distinct type for accessing the bits that ultimately make up object storage.
+// It is a distinct scoped enumeration type for accessing the bits that ultimately make up object storage.
 
```

Comment:
Let's do the refactoring in a separate patch. Otherwise it's really not clear which parts are just refactoring and which parts are actually related to the paper.